content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md

```diff
@@ -439,7 +439,287 @@ The onboard magnetometer of the Nicla Voice can be used to determine the board's
+  x_acc = x_acc_raw * ACCEL_SCALE_FACTOR;
+  y_acc = y_acc_raw * ACCEL_SCALE_FACTOR;
+  z_acc = z_acc_raw * ACCEL_SCALE_FACTOR;
+
+  // Convert raw gyroscope data to angular velocity expressed in °/s.
+  x_gyr = x_gyr_raw * GYRO_SCALE_FACTOR;
+  y_gyr = y_gyr_raw * GYRO_SCALE_FACTOR;
+  z_gyr = z_gyr_raw * GYRO_SCALE_FACTOR;
+  
+  // Print accelerometer data (expressed in m/s^2). 
+  Serial.print("x_acc:");
+  Serial.print(x_acc);
+  Serial.print(",");
+  Serial.print("y_acc:");
+  Serial.print(y_acc);
+  Serial.print(",");
+  Serial.print("z_acc:");
+  Serial.println(z_acc);
+
+  // Print gyroscope data (expressed in °/s). 
+  Serial.print("x_gyr:");
+  Serial.print(x_gyr);
+  Serial.print(",");
+  Serial.print("y_gyr:");
+  Serial.print(y_gyr);
+  Serial.print(",");
+  Serial.print("z_gyr:");
+  Serial.println(z_gyr);
+
+  delay(1000);
+}
+```
+
+Here you can find a step-by-step explanation of the code:
+
+First, the necessary libraries are included: 
+
+- `NDP.h` and `BMI270_Init.h` for the Nicla Voice board's basic functions and the IMU control.
+- Macros are defined for checking the status of the IMU; these macros allow the sketch to detect and handle sensor errors. 
+
+Next, user functions `ledBlueOn()`, `ledGreenOn()`, and `ledRedBlink()` definition: 
+
+- These functions allow the onboard LEDs to flash specific colors to indicate different states: blue for a successful match, green for an event, and red to indicate an error.
+
+Next, in the `setup()` function:
+
+- The serial communication is initialized at a baud rate of 115200.
+- The Nicla Voice board is initialized, and the LDO regulator (used for putting the board into power-saving modes) is disabled to avoid communication problems with the IMU. 
+- Error and event handlers are initialized.
+- NDP processor is initialized; this process includes populating the external Flash memory of the board with the NDP processor internal microcontroller firmware (`mcu_fw_120_v91.synpkg`), the NDP processor internal DSP firmware (`dsp_firmware_v91.synpkg`), and the ML model (`ei_model.synpkg`). 
+- The BMI270 sensor is initialized; this includes a software reset, loading the sensor configuration, and setting it into normal power mode with the accelerometer and gyroscope operational. 
+
+Finally, in the `loop()` function:
+
+- Memory is allocated for the sensor data; data is then read from the sensor and stored in this allocated space.
+- Raw sensor data is then parsed and extracted into raw accelerometer and gyroscope data. This data is represented as 16-bit signed integers ranging from -32 768 to 32 767.
+- Raw sensor data is converted into understandable and standard unit measurements; for the accelerometer, data is converted to meters per second squared, and for the gyroscope, data is converted to degrees per second. 
+- Converted accelerometer and gyroscope data are printed to the Serial Monitor, allowing one to observe sensor data in real time.

```

Comment:
Just a small rephrase: 
- Converted accelerometer and gyroscope data are printed on the Serial Monitor, allowing the user to observe sensor data in real-time.

Reply:
Suggestion added!